### ISSUE TITLE
Liaisons with digital publishing

### ISSUE THREAD

**Issue by iherman** [w3c/wcag#2547]
(This comment is on the [proposed charter](https://raw.githack.com/w3c/wcag/charter-2022/charter.html) as of 2022-07-14.)

The charter proposal includes:

> **[Publishing Working Group](https://www.w3.org/publishing/groups/publ-wg/)**
> Coordinate on accessibility guidelines that impact digital publishing.

As part of the liaison statements. There are some issues with this:

- The WG has been renamed [Audiobook WG](https://www.w3.org/publishing/groups/publ-wg/) (though with an unchanged WG URL), reflecting its somewhat narrower focus. The liaison statement should reflect this.
- There is no liaison statement to the [EPUB 3 WG](https://www.w3.org/publishing/groups/epub-wg/), in spite of the fact that the EPUB 3 series of documents have all major accessibility aspects and, in particular, the WG publishes an [EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/) Recommendation track document which heavily relies on WCAG.

I also wonder whether it is not more appropriate to have a more general liaison statement to the Digital Publishing work at W3C  (that may include the Digital Publishing Business and Community Groups) just like there is a statement towards the Mobile Web Initiative (although I wonder whether that Initiative is still official). I am not sure what the best official target is for that is, though.

Cc @wareid, @avneeshsingh, @tzviyasiegman

**Comment by avneeshsingh** [comment]
It may be a copy paste from WCAG 2.2 charter, when we had Publishing WG but no EPUB 3 WG (at 1).
It would be better to mention "Publishing groups at W3C", it would include all the existing publishing groups as well as new working groups that would be created in the lifetime of this charter.


1.
https://www.w3.org/2019/12/ag-charter

**Comment by michael-n-cooper** [comment]
Fixed to EPub WG: https://github.com/w3c/wcag/commit/98e2b9bf6ae41b05b1f4111994e0c9f24907796d

**Comment by iherman** [comment]
Sorry to be picky, @michael-n-cooper, but it should be EPUB WG, not EPub WG...
